index.js

```diff
@@ -215,6 +216,7 @@ class Queue extends EventEmitter {
       job.duration = job.endTime.getTime() - job.startTime.getTime();
       this.emit('failed', job, err);
     }
+    await this.notifyGroup(job);
```

Comment:
any reason not to do this after the update call?

Reply:
and then === 0 below.  just seems a bit cleaner

Reply:
Originally I had other tests that were failing and doing this with 1 made them work, but it looks like I fixed whatever that was, so this works with 0 now.